webots_ros2_epuck/setup.py

```diff
@@ -64,7 +66,7 @@
     tests_require=['pytest'],
     entry_points={
         'console_scripts': [
-            'driver = webots_ros2_epuck.driver:main',
+            'epuck_process = webots_ros2_epuck.epuck_node:main',
```

Comment:
```suggestion
            'epuck_node = webots_ros2_epuck.epuck_node:main',
```
I would keep it consistent